### DISCUSSION TITLE
React Strict Mode - distinguish between first and second render

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#7946] [Ideas]
Not sure if this is possible but for React strict mode it’d be really useful if Replay had a similar way to denote if a logpoint is the second render / reducer call or not.
I’m assuming you know what I’m talking about but just for reference this is what shows when React devtools instruments console.X
Specifically asking about this because all new Next.js apps are opted into strict mode. The new router we’re working on also fully integrates concurrent features so we always have strict mode enabled while developing on Next.js as well. (edited) 
![image](https://user-images.githubusercontent.com/254562/194905285-1b7ea393-81bf-4d2b-b32d-ef6d83f84dbc.png)

**Comment by bvaughn** [comment]
What was meant by "distinguish" ? Is the ask to preserve the different styling for console logs or something more global?

Considering the console styling is accomplished by overriding built-in console methods, it should be possible for us to detect and/or do something similar.

Related code pointers:
* https://github.com/facebook/react/blob/main/packages/shared/ConsolePatchingDev.js
* https://github.com/facebook/react/blob/main/packages/react-devtools-shared/src/backend/console.js
